src/Types/Redeemer.purs

```diff
@@ -0,0 +1,18 @@
+module Types.Redeemer
```

Comment:
This module's name would suggest (to me at least) that it defines a `Redeemer` type, but it only defines and exports `unitRedeemer`. I think that the `unitReedemer` function (which is only for convenience) can just go in `Helpers` or something and this module can be deleted

Reply:
I think I refactored quite a few things, moving the datatype `Redeemer` under `Types.Datum` as they're closely related, feel free to reconcile with these changes in a manner you see fit.